### ISSUE TITLE
Not working on UPBGE 0.2.5b

### ISSUE THREAD

**Issue by CGI-Fox** [IzaZed/Uchronian-Logic-UPBGE-Logic-Nodes#50]
This is the error message that it sends:
![Sin título](https://user-images.githubusercontent.com/84427482/118745526-5d627880-b824-11eb-8f67-44feb4dbfe77.png)

**Comment by IzaZed** [comment]
This is a known one, the current version depends on quite a lot of the 0.3.0 functionality

there is a release specifically for 0.2.5, but it's a little outdated
I'll get the 0.2.5 version up and running with the newer features soon though, so I'll leave this open

**Comment by CGI-Fox** [comment]
> This is a known one, the current version depends on quite a lot of the 0.3.0 functionality
> 
> there is a release specifically for 0.2.5, but it's a little outdated
> I'll get the 0.2.5 version up and running with the newer features soon though, so I'll leave this open

Thank you a lot, I wanna make a big ass code in one object, do you know another way to mae the logic brick more organized?

**Comment by IzaZed** [comment]
Not really, there is a another addon that wraps the logic bricks into nodes, but I did not test it on 0.2.5 🤷🏼‍♂️

**Comment by CGI-Fox** [comment]
> Not really, there is a another addon that wraps the logic bricks into nodes, but I did not test it on 0.2.5 🤷🏼‍♂️

Either works. I will wait for you release, Thank you

**Comment by igame3dbill** [comment]
This doesn't work in 0.2.5b and the 0.8.8 release didn't show any logic node editor in the editor list.

**Comment by pozitigor1** [comment]
> This doesn't work in 0.2.5b and the 0.8.8 release didn't show any logic node editor in the editor list.

maybe its a little bit too late, but in the node editor there appears a plus button to the left of the material button
![UPBGE_  D__Blender_Games_OldHouse_OldHouseGame blend  26 12 2021 19_36_20 (2)](https://user-images.githubusercontent.com/61246816/147411391-21e9d8a0-b015-421b-a103-82393ab1c30e.png)

**Comment by HexOverride** [comment]
did the version ever release?
